cclib/parser/serenityparser.py

```diff
@@ -52,6 +53,17 @@ def after_parsing(self):
 
         super().after_parsing()
 
+    def convert_to_spin(self, symbol):
+        if symbol == "a":
+            return 0
+        elif symbol == "b":
+            return 1
+        else:
+            self.logger.warning(
```

Comment:
Since you're making this unrecoverable by throwing `StopParsing`, this could probably be an error instead of a warning.